### ISSUE TITLE
Suggestion: [Aggrocolor]

### ISSUE THREAD

**Issue by Breakums** [Aviana/LunaUnitFrames#744]
As I understand, aggrocolor has been reworked with blizz's api to give a true aggro indicator. I was using the original tag as a targeted indicator for predictive healing and pvp.  Can there be another tag that uses the old mechanic?

**Comment by lvangh** [comment]
Yes, this change has made healing difficult for prediction because the threat table isn't updating as fast as the simple Target/Target check from the prior version. So you don't know who will get hit just from raid frames until it's possibly too late.

**Comment by Aviana** [comment]
The targettarget check has been re-added under squares in v4000. I will check if tags get it aswell at a later point.
